plugins-ui/src/modules/marketplace/components/marketplace-filters/MarketplaceFilters.tsx

```
@@ -1,28 +1,133 @@
+const sortLabels = Object.keys(sortOptions);
 
 const MarketplaceFilters = ({
+  categories,
   viewFilter,
-  onChange,
+  onViewChange,
+  filters,
+  onFiltersChange,
 }: MarketplaceFiltersProps) => {
+  const categoryNames = categories.map(c => c.name);
+  const defaultCategoryName = (categories.find(c => c.id === filters.categoryId) || {}).name || "";
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)